opentelemetry-api/src/opentelemetry/trace/ids_generator.py

```diff
@@ -0,0 +1,33 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+
+class IdsGenerator(abc.ABC):
+    @abc.abstractmethod
+    def generate_span_id(self) -> int:
+        """Get a new random span ID.
```

Comment:
Don't think `random` is a feature of this interface, that's the implementation

Reply:
I see what you're saying, makes sense, will make the change!

Reply:
I elected to say "Get a new span ID." instead of "Get a new unique span ID." because I don't want to assume how random implementations will be.